cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -1912,7 +1917,7 @@ end subroutine strain_rates
 ! author: JF Lemieux, ECCC
 ! Nov 2021
 
-      subroutine strain_rates_T (nx_block,   ny_block, &
+      subroutine strain_rates_Tdtsd (nx_block,   ny_block, &
```

Comment:
what does dtsd signify?
Oh, maybe div, tension, shear, delta - got it - please add this to the comment at the top of the routine